ecosystem/typescript/sdk/examples/typescript/multisig_managed_coin.ts

```diff
@@ -0,0 +1,294 @@
+/* eslint-disable no-console */
+
+import dotenv from "dotenv";
+dotenv.config();
+
+import { AptosClient, AptosAccount, FaucetClient, BCS, TxnBuilderTypes, Types, MaybeHexString, HexString } from "aptos";
+import { sha3_256 as sha3Hash } from "@noble/hashes/sha3";
+import { FAUCET_URL, NODE_URL, fungibleStore } from "./common";
+import assert from "assert";
+
+const { AccountAddress, EntryFunction, MultiSig, MultiSigTransactionPayload, TransactionPayloadMultisig } =
+  TxnBuilderTypes;
+
+// Step 0: After publishing the example code with the example code under any `MODULE_ADDR` using CLI or any sdk.
+const MODULE_ADDR = process.env.MODULE_ADDR;
+const ASSET_SYMBOL = "MEME";
+
+/**
+ * This code example demonstrates how to use framework mulltisig account module to manage fungible asset with exmaple move code.
+ */
+(async () => {
+  const client = new AptosClient(NODE_URL);
+  const faucetClient = new FaucetClient(NODE_URL, FAUCET_URL);
+
+  console.log(`node url: ${NODE_URL}`);
+  console.log(`faucet url: ${FAUCET_URL}`);
+
+  // Create and fund 3 accounts that will be the owners of the multisig account where owner1 would be the one that publish the move module later.
+  // :!:>section_1
+  const owner1 = new AptosAccount();
+  const owner2 = new AptosAccount();
+  const owner3 = new AptosAccount();
+  await faucetClient.fundAccount(owner1.address(), 100_000_000);
+  await faucetClient.fundAccount(owner2.address(), 100_000_000);
+  await faucetClient.fundAccount(owner3.address(), 100_000_000);
+  console.log(`owner1: ${owner1.address()}`);
+  console.log(`owner2: ${owner2.address()}`);
+  console.log(`owner3: ${owner3.address()}`);
+  // <:!:section_1
+
+  // Step 1: Setup a k-of-n (2-of-3 here) multisig account
+  // ===========================================================================================
+  // :!:>section_2
+  // Find the next multisig account address.
+  const payload: Types.ViewRequest = {
+    function: "0x1::multisig_account::get_next_multisig_account_address",
+    type_arguments: [],
+    arguments: [owner1.address().hex()],
+  };
+  const multisigAddress = (await client.view(payload))[0] as string;
+
+  const createMultisigManagedCoin = await client.generateTransaction(owner1.address(), {
+    function: MODULE_ADDR + "::multisig_managed_coin::initialize",
+    type_arguments: [],
+    arguments: [
+      [owner2.address().hex(), owner3.address().hex()],
+      2,
+      ["description"],
+      [BCS.bcsSerializeStr("The multisig account for Meme")],
+      0,
+      "meme coin",
+      ASSET_SYMBOL,
+      8,
+      "http://meme.xyz/favicon.ico",
+      "http://meme.xyz",
+      [true, true, true],
+    ],
+  });
+  // <:!:section_2
+  await client.generateSignSubmitWaitForTransaction(owner1, createMultisigManagedCoin.payload);
```

Comment:
a bit confused of why you `generateTransaction` on line 52, and then pass that generated raw transaction.payload to `generateSignSubmitWaitForTransaction`.. why not just
```
await client.generateSignSubmitWaitForTransaction(owner1,{
    function: MODULE_ADDR + "::multisig_managed_coin::initialize",
    type_arguments: [],
    arguments: [
      [owner2.address().hex(), owner3.address().hex()],
      2,
      ["description"],
      [BCS.bcsSerializeStr("The multisig account for Meme")],
      0,
      "meme coin",
      ASSET_SYMBOL,
      8,
      "http://meme.xyz/favicon.ico",
      "http://meme.xyz",
      [true, true, true],
    ],
  }
```

Reply:
cuz I follow what `multisig_managed_coin.ts` did.. I don't even know that it can be that concise. lol

Reply:
I think `generateSignSubmitWaitForTransaction` needs `TransactionPayload` but not `EntryFunctionPayload`. Are you sure this works?